Sources/Plasma/PubUtilLib/plNetGameLib/Private/plNglAuth.cpp

```diff
@@ -2657,7 +2641,7 @@ bool LoginRequestTrans::Send () {
         kCli2Auth_AcctLoginRequest,
         m_transId,
         clientChallenge,
-        (uintptr_t) s_accountName,
+        (uintptr_t) s_accountName.ToWchar().GetData(),
```

Comment:
This will return a `plStringBuffer` temporary, which will disappear after the call is finished.  Have you made sure this is copied before the call is finished?